src/startHtml.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * start_html
+ *
+ * This plugin redacts all HTML tags and content in a start_html level string,
+ * such that the translator only sees the relevant text to translate.
+ *
+ * @example
+ * Source: <label
+ *          style="margin: 0px; padding: 2px;
+ *              line-height: 1; font-size: 40px;
+ *              overflow: hidden; word-wrap: break-word;
+ *              color: rgb(51, 51, 51);
+ *              max-width: 320px; width: 320px; height: 45px;
+ *              position: absolute; left: 0px; top: 15px;
+ *              text-align: center;"
+ *           id="label3">
+ *           Success!
+ *         </label>
+ * Redacted: [][0]Success![][1]
+ */
+
+let redact;
+const HTML_TAG_RE = /<\/{0,1}[a-zA-Z]+(>|.*?[^?]>)/;
```

Comment:
Being a regex newb: is this like a universal pattern you found?

Reply:
kind of! found the character pattern, then added a check for forward slash to include a closing tag. regexpal.com is what I use to understand expressions

Reply:
Got it! 👍

Reply:
One thing I would note here is that there is - [somewhat infamously](https://stackoverflow.com/a/1732454/1810460) - no way to _actually_ parse HTML with regex. An approximation like this can work well enough for purposes of something simple like redaction, but we should make sure to be aware of its limitations